### PR TITLE
linea mini-hack

### PR DESCRIPTION
**Pull request by emilyJLin95**

### CODE REVIEW

File: docs/build-on-linea/hackathons.md [hunk 1]
```diff
@@ -21,36 +21,79 @@ And be featured on [X](https://twitter.com/lineabuild), [Linea Farcaster](https:
...
+- **Deploy a smart contract**
+  - Follow our [step-by-step quickstart on how to deploy and verify a smart contract](/build-on-linea/quickstart) with a variety of tools like Hardhat, Foundry, and more!
+- **Get access to Verax private beta**
+  - Send your wallet address in our [Telegram chat](https://t.me/+rI-iPLacQXQ5MDVh) to get access to Verax private beta! Here's a [quick tutorial](https://docs.ver.ax/verax-documentation/developer-guides/tutorials/from-a-schema-to-an-attestation) on how to leverage Verax.
+- [**Walkthroughs and use cases**](https://youtube.com/playlist?list=PLJ06SwdM0bLrA-3EGRji4W0QI8fyA8PyW&si=vQsXrtFVUsXhygJ0)
+- **Get help**
+  - Join our [developer chat](https://t.me/+rI-iPLacQXQ5MDVh) where you can connect with teams across MetaMask, Linea, Phosphor, Verax, Infura, and more!
 
-Use our [faucet or bridge to get mainnet or testnet eth](/use-mainnet/fund). Otherwise, [ping us in our Telegram channel](https://t.me/+rI-iPLacQXQ5MDVh) or find us IRL at the hackathon to request test ETH directly!
+## Monthly Mini-Hack
```
Comment (jlwllmr) [inline docs/build-on-linea/hackathons.md:34]:
```suggestion
## Monthly mini-hack
```

File: docs/build-on-linea/hackathons.md [hunk 2]
```diff
@@ -21,36 +21,79 @@ And be featured on [X](https://twitter.com/lineabuild), [Linea Farcaster](https:
...
+- Incorporating [partner tooling](https://docs.linea.build/build-on-linea/tooling) to make more robust and interesting examples
+- A frog is somewhere in your submission
+
+_Judging criteria_
+- UX/UI
+- Utility
+- Technicality
+- Market readiness
+
+## IRL Hackathons
```
Comment (jlwllmr) [inline docs/build-on-linea/hackathons.md:88]:
```suggestion
## IRL hackathons
```

File: docs/build-on-linea/hackathons.md [hunk 3]
```diff
@@ -21,36 +21,79 @@ And be featured on [X](https://twitter.com/lineabuild), [Linea Farcaster](https:
...
+
+### EthVietnam
+
+**Build a defi, deso, gaming, or frog themed dapp on Linea ($5k pool prize)**
+
+We’re looking for the novel ideas expanding the adoption of Web3 and want to connect with hackers, giving them exclusive invites to our Linea Builders Club, where you can find tech talks by our ZK cryptographers, private mini hack opportunities, educational programs, and more!
+
+We're judging based on technicality, utility, UX, and market readiness - bonus points if you add a frog-theme or use any partner tooling (https://docs.linea.build/build-on-linea/tooling)
```
Comment (jlwllmr) [inline docs/build-on-linea/hackathons.md:96]:
```suggestion
We're judging based on technicality, utility, UX, and market readiness - bonus points if you add a frog theme or use any [partner tooling](https://docs.linea.build/build-on-linea/tooling)
```

File: docs/build-on-linea/hackathons.md [hunk 4]
```diff
@@ -60,33 +103,34 @@ Build a defi, social, or gaming app to build the next big use case on Linea! All
 
 *Judging criteria*
 
+- Technicality
 - Utility
-- Innovation
-- Technical sophistication
+- Market Readiness
```
Comment (jlwllmr) [inline docs/build-on-linea/hackathons.md:108]:
```suggestion
- Market readiness
```

File: docs/build-on-linea/hackathons.md [hunk 5]
```diff
@@ -60,33 +103,34 @@ Build a defi, social, or gaming app to build the next big use case on Linea! All
...
 
-You can find example use cases [here](https://explorer.ver.ax/linea).
+**Build any dapp on Linea ($2k)**
 
-*Requirements*
+All you have to do to qualify is deploy a smart contract on Linea!
 
-- Description of how you leveraged Verax in your dapp
-- Link to a short video walking through a demo of your project and how you built it
+We're judging based on technicality, utility, and market readiness - bonus points if you add a frog-theme or use any partner tooling (https://docs.linea.build/build-on-linea/tooling)
```
Comment (jlwllmr) [inline docs/build-on-linea/hackathons.md:120]:
```suggestion
We're judging based on technicality, utility, and market readiness - bonus points if you add a frog theme or use any [partner tooling](https://docs.linea.build/build-on-linea/tooling)
```

File: docs/build-on-linea/hackathons.md [hunk 6]
```diff
@@ -60,33 +103,34 @@ Build a defi, social, or gaming app to build the next big use case on Linea! All
...
+- Any bonus point integrations you added on your Github README
+- Link to a short video walking through a demo of your project and how you built it
 
-### EthTaipei
+*Judging criteria*
 
-Bounty information under construction. Sign up for the EthTaipei hackathon [here](https://taikai.network/ethtaipei/hackathons/hackathon-2024)!
+- Technicality
+- Utility
+- Market Readiness
```
Comment (jlwllmr) [inline docs/build-on-linea/hackathons.md:134]:
```suggestion
- Market readiness
```

**Comment by emilyJLin95** [comment]
@jlwllmr made changes! Can you help approve and merge?

**Comment by emilyJLin95** [comment]
@jlwllmr made changes! Need to update for the vietnam hackathon - we can make necessary changes afterwards in case there is more to make

**Pull request merged**